Doc/whatsnew/3.12.rst

```diff
@@ -159,7 +159,9 @@ Deprecated
   and tailor them to your needs.
   (Contributed by Erlend E. Aasland in :gh:`90016`.)
 
-* The throw\(type, value, traceback\) signature is now deprecated. (:gh:`89874`.)
+* The 3-arg signatures (type, value, traceback) of :meth:`~coroutine.throw`
+  and :meth:`~generator.throw` are deprecated and may be removed in a future
+  version of Python. Use the single-arg versions of these functions instead. (:gh:`89874`.)
```

Comment:
```suggestion
  version of Python. Use the single-arg versions of these functions instead.
  (Contributed by Ofey Chan in :gh:`89874`.)
```

Reply:
You can also add yourself to https://github.com/python/cpython/blob/main/Misc/ACKS